decentralizediso20022/src/main/java/com/centiglobe/decentralizediso20022/application/ValidationService.java

```diff
@@ -29,7 +29,7 @@ public static void validateHeaderTo(BusinessAppHdrV02 header, String truststore,
         String domain = header.getTo().getFIId().getFinInstnId().getNm();
 
         if (truststore != null && pwd != null) {
-            checkSSLCertificate("https://" + domain, truststore, pwd);
+            checkSSLCertificate("https://" + domain + ":8443", truststore, pwd);
```

Comment:
The port should probably be defined in a .properties file.

Reply:
I added the port as an argument.